analytics/views.py

```diff
@@ -244,7 +244,8 @@ def get_chart_data(request: HttpRequest, user_profile: UserProfile, chart_name:
         if end is None:
             end = max(last_successful_fill(stat.property) or
                       datetime.min.replace(tzinfo=timezone_utc) for stat in stats)
-        if start > end:
+
+        if start > end and (timezone_now() - start > timedelta(hours=24)):
```

Comment:
@timabbott I am wondering whether we should replace this with the following conditional. Let me call it approach 2 for easy reference. 

```
if timezone_now() - end > timedelta(hours=24)
```

So approach 2 will instantly throw an error if our cronjob stops running and someone opens /stats page of any realm.

The current approach in the PR throws an error only if someones open the stats page of a realm that was created after the cron job stopped running. Also the newly created realm should be older than 24hours for the error to be shown. Stats pages of realms that were created before the cron job stopped would continue working even if the cron job stops working.